### ISSUE TITLE
Simple markdown explanations

### ISSUE THREAD

**Issue by drphilmarshall** [DrMud/SentimentHackPad#1]
Let's take a few mins to add in some text cells explaining to our future selves what we are doing, shall we? This'd be a good place to supply the links we used as well, for future reference - https://textblob.readthedocs.io should definitely be one of them

**Comment by piccolomud** [comment]
yeah, I think I hurriedly added some text before the first reveal on Thursday. But I'll add the textblob link.

**Comment by drphilmarshall** [comment]
Cool, thanks Madhura :-) My future self is grateful too!
